src/core/database.ts

```diff
@@ -29,9 +34,10 @@ export interface DropBody {
 
 // Encodable database is the same as database but procedures.returns can be an empty array
 export type EncodeableDatabase = Omit<Database, 'tables' | 'procedures' | 'foreign_calls'> & {
-  tables: ReadonlyArray<CompiledTable>,
-  procedures: ReadonlyArray<CompiledProcedure>,
-  foreign_calls: ReadonlyArray<CompiledForeignProcedure>,
+  tables: ReadonlyArray<CompiledTable>;
+  procedures: ReadonlyArray<CompiledProcedure>;
+  foreign_calls: ReadonlyArray<CompiledForeignProcedure>;
+  type?: 'encodeable_database';
```

Comment:
Why? This feels quite weird to put here because `type` is not a database property that kwild will accept.